src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+module.exports = function(options, undef) {
+  'use strict';
+
+  const TAU = 2 * Math.PI,
+        //TAU = options.PConstants.TAU,
+        lerp = function(start, stop, amt) {
```

Comment:
Once that it's in place, I'll change that to it.
For now I need users to test whether it even runs!